<?xml version="1.0"?>
<launch>
  <arg name="manager_name" default="move_ukf_nodelet_manager" />
  <arg name="manager_threads" default="4" />

  <node pkg="nodelet" type="nodelet" name="$(arg manager_name)" args="manager" output="screen">
    <param name="num_worker_threads" value="$(arg manager_threads)" />
  </node>

  <node pkg="nodelet" type="nodelet" name="bottom_optical_flow_integrator"
    args="load plugin_lecture/plugin_nodelet_talker $(arg manager_name)">
    <remap from="chatter" to="chatter_alt" />
    <param name="sensor_frame" value="optical_flow_bottom_link" />
    <param name="body_frame" value="base_link" />
  </node>


</launch>